<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>97</y>
      <width>471</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="table">
     <item row="1" column="2">
      <widget class="QPlainTextEdit" name="btn1_2">
       <property name="font">
        <font>
         <pointsize>43</pointsize>
        </font>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPlainTextEdit" name="btn0_1">
       <property name="font">
        <font>
         <pointsize>43</pointsize>
        </font>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPlainTextEdit" name="btn0_2">
       <property name="font">
        <font>
         <pointsize>43</pointsize>
        </font>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPlainTextEdit" name="btn1_0">
       <property name="font">
        <font>
         <pointsize>43</pointsize>
        </font>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPlainTextEdit" name="btn2_1">
       <property name="font">
        <font>
         <pointsize>43</pointsize>
        </font>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPlainTextEdit" name="btn2_2">
       <property name="font">
        <font>
         <pointsize>43</pointsize>
        </font>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPlainTextEdit" name="btn2_0">
       <property name="font">
        <font>
         <pointsize>43</pointsize>
        </font>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPlainTextEdit" name="btn0_0">
       <property name="font">
        <font>
         <pointsize>43</pointsize>
        </font>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="plainText">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPlainTextEdit" name="btn1_1">
       <property name="font">
        <font>
         <pointsize>43</pointsize>
        </font>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="Previous">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>377</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Previous</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Next">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>377</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Cancel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>377</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="First">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>427</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>First</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Last">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>427</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Last</string>
    </property>
   </widget>
   <widget class="QLabel" name="moveNum">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>47</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="numAll">
    <property name="geometry">
     <rect>
      <x>96</x>
      <y>10</y>
      <width>461</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
